### PR TITLE
Version 3.8

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#349 #350
#### What does this implement/fix? Explain your changes.
Version 3.8

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@alirezazolanvari @sadrasabouri Hi folks
Please complete this review by **Wednesday- 2023-02-01 - 11:59 AM GMT+3:30**
**Notice:** Only review, don't need to merge

**Review by sadrasabouri — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,51 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+	26. Consonni & Todeschini I
+	27. Consonni & Todeschini II
+	28. Consonni & Todeschini III
+	29. Consonni & Todeschini IV
+	30. Consonni & Todeschini V
+### Changed
+- `relabel` method sort bug fixed
+- `README.md` modified
+- Document modified
+- Test system modified
```
Comment (alirezazolanvari) [inline CHANGELOG.md:52]: Shouldn't we mention the changes in `compare` overall benchmarks and overall benchmarks weights?

Comment (sepandhaghighi) [reply CHANGELOG.md:52]: You are right!

- `Compare` overall benchmarks default weights updated

Good?

Comment (sepandhaghighi) [reply CHANGELOG.md:53]: fixed in https://github.com/sepandhaghighi/pycm/pull/481/commits/f2c225a008d946de2e9be0a50f0d9a827b54f92f

**Review by alirezazolanvari — APPROVED** [review]

**Pull request merged**